docs/rules/consistent-return.md

```diff
@@ -21,6 +21,10 @@ Here, one branch of the function returns `true`, a Boolean value, while the othe
 
 This rule is aimed at ensuring all `return` statements either specify a value or don't specify a value.
 
+It excludes constructors which, when invoked with the `new` operator, return the instantiated object if another object is not explicitly returned.  This rule judges from following conditions whether or not the function is a constructor:
```

Comment:
Good catch.  Thanks!  Copy/Paste error on my part where the second condition of `no-invalid-this` doesn't apply in this case.  Think it's worth keeping a bulleted list to be consistent with `no-invalid-this` or just collapse it into a single sentence?